modules/nf-core/cellranger/count/templates/cellranger_count.py

```diff
@@ -58,13 +59,19 @@ def chunk_iter(seq, size):
 run(
     # fmt: off
     [
-        "cellranger", "count",
-        "--id", "${prefix}",
-        "--fastqs", str(fastq_all),
-        "--transcriptome", "${reference.name}",
-        "--localcores", "${task.cpus}",
-        "--localmem", "${task.memory.toGiga()}",
-        *shlex.split("""${args}""")
+        "cellranger",
+        "count",
+        "--id",
+        "${prefix}",
+        "--fastqs",
+        str(fastq_all),
+        "--transcriptome",
+        "${reference.name}",
+        "--localcores",
+        "${task.cpus}",
+        "--localmem",
+        "${task.memory.toGiga()}",
+        *shlex.split("""${args}"""),
```

Comment:
@edmundmiller, does ruff not respect the `# fmt: off` comments? 
I added those comments, because it's more readable to have the parameter name and the associated parameter in the same line. 

With black, this worked just fine

Reply:
Is there a reason you don't format it as
```
"--transcriptome ${reference.name}"
```

Reply:
you need to do it outside the expression: https://docs.astral.sh/ruff/formatter/#format-suppression

Reply:
> `"--transcriptome ${reference.name}"`

I don't think this would work. In that case it would be treated as a single argument, i.e. the same as if you pass
```
cellranger count "--transcriptome ${reference.name}"
```
in bash.